pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -368,6 +368,8 @@ func (vm *volumeManager) WaitForAttachAndMount(pod *v1.Pod) error {
 			return nil
 		}
 
+		vm.addUnmountedVolumesStatus(unmountedVolumes)
```

Comment:
silly question: where do these status' get consumed/exposed?

Reply:
The status being printed in the pod events ('kubectl describe pod'). The whole point of the patch is to calm the user down when the pod event log contains mount time-out messages. In the case the pod is being set up, ie. the recursive chown takes long time there's no way to speed it up and the mount needs to be re-tried again (the volume is actually mounted on the system but must not be mounted to the pod yet) so the time-out does not mean anything wrong happened in such case.

The pod events are easily accessible so it looked like a good place to expose the information.

Reply:
Thanks for the explanation, sounds good to me